code/index.html

```diff
@@ -3,28 +3,142 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
+    <title>Business site</title>
+
+          <!-- google fonts -->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Courgette&family=Ma+Shan+Zheng&family=Syncopate&display=swap" rel="stylesheet">
+          <!-- My CSS file -->
+    <link rel="stylesheet" href="./style.css">
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+         <!--head-section-->
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <div class="container">
+      <img class="logo" src="./img/relaxlogobrons.png" alt="logo front">     
+      <a href="#book"><button class="button">Booking</button></a>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+        <!--Background video-->
+
+      <video class="bg-video" autoplay muted loop playsinline>
+        <source src="./videos/relax.mp4" type="video/mp4">
+        <p>This browser doesn't support video...</p>
+      </video> 
+      <h1 class="slogan">Treat yourself<br> 
+      like you treat others</h1>
+    </div>
+
+           <!-- Signup form -->
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <section class="form-and-faq-container" id="book">
+      <form class="form" action="http://httpbin.org/anything" method="POST">
+        <h2 class="form-header">Sign up for the retreat</h2>
+        <div class="contact-booking">
+        <div class="contact-details">
+          <h3 class="form-title">contact details</h3>
+          <label for="name">Your Name:
+            <input type="text" name="name" required placeholder="Name">
+          </label>
+          <label for="email" > your Email:
+            <input type="email" name="email" required placeholder="Email">
+          </label>
+          <label for="Phonenumber">Your Phone number:
+            <input type="text" name="phonenumber" required placeholder="Phone number">
```

Comment:
for the phone number you can use instead `<input type="number" />` so that the user cannot type any text :)